acmw-db-app/pages/api/student/[pid].js

```diff
@@ -0,0 +1,33 @@
+import pgQuery from '../../../postgres/pg-query.js';
+
+// GET /api/student/totalUniqueMembers
+async function totalUniqueMembers () {
```

Comment:
needs to incorporate “for a given academic year” i.e. check that they attended a meeting in AU__ or SP__